c++/src/security/InMemoryKeystore.cc

```diff
@@ -0,0 +1,247 @@
+// Copyright 2010-present vivo, Inc. All rights reserved.
```

Comment:
Please note that we have a license check which does not allow custom modification to the license header in this repo. I'm not sure if this violates any ASF rule to add something like this. @dongjoon-hyun Do you have any concern?